lib/smart_answer_flows/register-a-birth/outcomes/oru_result.govspeak.erb

```diff
@@ -203,8 +203,8 @@
 
   Service | Fee
   -|-
-  Register a birth | £105
-  Copy of a birth registration certificate | £65
+  Register a birth | <%= format_money(reg_data_query.register_a_birth_fees.register_a_birth) %>
+  Copy of a birth registration certificate | <%= format_money(reg_data_query.register_a_birth_fees.copy_of_birth_registration_certificate) %>
```

Comment:
@chrisroos @pmanrubia @floehopper should can we consider adding these to the consular fees table partial?